src/app/scheduler/components/CourseCard.tsx

```diff
@@ -0,0 +1,89 @@
+import { AddIcon, CloseIcon, InfoOutlineIcon } from '@chakra-ui/icons';
+import { Box, Text, Flex, VStack, Checkbox, IconButton, BackgroundProps } from '@chakra-ui/react';
+import { useCallback } from 'react';
+import { Link } from 'react-router-dom';
+
+import { Term } from '../../../shared/fetchers';
+import { useSavedCourses } from '../../../shared/hooks/useSavedCourses';
+
+export type CourseCardProps = {
+  term: Term;
+  subject: string;
+  title: string;
+  code: string;
+  color: BackgroundProps['bg'];
+  pid: string;
+  selected: boolean;
+  handleChange: Function;
+  handleBookmarkClick: Function;
+};
+
+export function CourseCard({
+  term,
+  subject,
+  title,
+  code,
+  color,
+  pid,
+  selected,
+  handleChange,
+  handleBookmarkClick,
+}: CourseCardProps): JSX.Element {
+  const { contains } = useSavedCourses();
+  const courseIsSaved = contains(pid, term);
```

Comment:
This should be removed as well. The reason being the same as the other removals of hooks.